src/patternfly/layouts/Grid/grid.scss

```diff
@@ -79,4 +86,156 @@
+        justify-items: start;
+      }
+
+      &.pf-m-justify-items-center#{$breakpoint-name} {
+        justify-items: center;
+      }
+
+      &.pf-m-justify-items-end#{$breakpoint-name} {
+        justify-items: end;
+      }
+
+      &.pf-m-justify-items-stretch#{$breakpoint-name} {
+        justify-items: stretch;
+      }
+
+      // align-items
+      &.pf-m-align-items-start#{$breakpoint-name} {
+        align-items: start;
+      }
+
+      &.pf-m-align-items-center#{$breakpoint-name} {
+        align-items: center;
+      }
+
+      &.pf-m-align-items-end#{$breakpoint-name} {
+        align-items: end;
+      }
+
+      &.pf-m-align-items-stretch#{$breakpoint-name} {
+        align-items: stretch;
+      }
+
+      // justify-content
+      &.pf-m-justify-content-start#{$breakpoint-name} {
+        justify-content: start;
+      }
+
+      &.pf-m-justify-content-center#{$breakpoint-name} {
+        justify-content: center;
+      }
+
+      &.pf-m-justify-content-end#{$breakpoint-name} {
+        justify-content: end;
+      }
+
+      &.pf-m-justify-content-stretch#{$breakpoint-name} {
+        justify-content: stretch;
+      }
+
+      &.pf-m-justify-content-space-around#{$breakpoint-name} {
+        justify-content: space-around;
+      }
+
+      &.pf-m-justify-content-space-between#{$breakpoint-name} {
+        justify-content: space-between;
+      }
+
+      &.pf-m-justify-content-space-evenly#{$breakpoint-name} {
+        justify-content: space-evenly;
+      }
+
+      // align-content
+      &.pf-m-align-content-start#{$breakpoint-name} {
+        align-content: start;
+      }
+
+      &.pf-m-align-content-center#{$breakpoint-name} {
+        align-content: center;
+      }
+
+      &.pf-m-align-content-end#{$breakpoint-name} {
+        align-content: end;
+      }
+
+      &.pf-m-align-content-stretch#{$breakpoint-name} {
+        align-content: stretch;
+      }
+
+      &.pf-m-align-content-space-around#{$breakpoint-name} {
+        align-content: space-around;
+      }
+
+      &.pf-m-align-content-space-between#{$breakpoint-name} {
+        align-content: space-between;
+      }
+
+      &.pf-m-align-content-space-evenly#{$breakpoint-name} {
+        align-content: space-evenly;
+      }
+
+      &.pf-m-grid-auto-flow-column#{$breakpoint-name} {
+        grid-auto-flow: column;
+      }
+
+      &.pf-m-grid-auto-flow-row#{$breakpoint-name} {
+        grid-auto-flow: row;
+      }
+
+      &.pf-m-grid-auto-flow-dense#{$breakpoint-name} {
+        grid-auto-flow: dense;
+      }
+
+      // justify-self
+      .pf-m-justify-self-start#{$breakpoint-name} {
+        justify-self: start;
+      }
+
+      .pf-m-justify-self-center#{$breakpoint-name} {
+        justify-self: center;
+      }
+
+      .pf-m-justify-self-end#{$breakpoint-name} {
+        justify-self: end;
+      }
+
+      .pf-m-justify-self-stretch#{$breakpoint-name} {
+        justify-self: stretch;
+      }
+
+      // align-self
```

Comment:
nit - you do `-start`, `-center`, `-end`, `-stretch` in all of the other blocks, but have swapped `-end` and `-center` here

Reply:
ah thanks I'll fix it